examples/ConvertPyImageJDataset.ipynb

```diff
@@ -0,0 +1,1153 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cffe4de0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys, os\n",
+    "!conda install --yes --prefix {sys.prefix} -c conda-forge openjdk=8\n",
+    "os.environ['JAVA_HOME'] = os.sep.join(sys.executable.split(os.sep)[:-2] + ['jre'])\n",
+    "!{sys.executable} -m pip install spatial-image-multiscale matplotlib zarr pyimagej"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "a3a9d3cf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from spatial_image_multiscale import to_multiscale\n",
+    "from spatial_image import is_spatial_image, to_spatial_image\n",
+    "import imagej\n",
+    "import zarr"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "edf6a61b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Failed to guess the Java version.\n",
+      "Picked up _JAVA_OPTIONS: -Dawt.useSystemAAFontSettings=gasp\n",
+      "WARNING: An illegal reflective access operation has occurred\n",
+      "WARNING: Illegal reflective access by net.imagej.patcher.LegacyInjector (file:/home/matt/.jgo/net.imagej/imagej/5f34b9963e566d47fe91383d53a5332bfc13df00c5d2f4bd13e2ea10b8f5fb2e/ij1-patcher-1.2.2.jar) to method java.lang.ClassLoader.findLoadedClass(java.lang.String)\n",
```

Comment:
Looks like you are using Java 11. Can you tell setup-java to use Java 8 instead? It's not a huge deal, Java 11 should work, but you'll avoid these reflection errors. And Java 17 will unfortunately have problems with the bytecode patching that ImageJ2 does around the original ImageJ.